run_test.go

```diff
@@ -60,15 +59,15 @@ func TestAddRlimits(t *testing.T) {
 				for _, rlimit := range rlimits {
 					if rlimit.Type == "RLIMIT_FSIZE" {
 						if rlimit.Hard != 4096 {
-							return fmt.Errorf("expected spec to have %#v hard limit set to %v but got %v", rlimit.Type, 4096, rlimit.Hard)
+							return errors.Errorf("expected spec to have %#v hard limit set to %v but got %v", rlimit.Type, 4096, rlimit.Hard)
```

Comment:
lint is unhappy with this line.  Could it be errors doesn't handle `%#v`?

Reply:
Fixed, was a bad (missing) import. repushed